Makefile

```diff
@@ -148,6 +148,11 @@ go-test: export TEST_ASSET_ETCD = $(ETCD)
 go-test: $(KUBECTL) $(KUBE_APISERVER) $(ETCD) ## Run go tests
 	go test ./...
 
+.PHONY: test-cover
```

Comment:
the controller tests usually fails when you just run go test with the env variables set by make test

Reply:
/lgtm cancel

Reply:
@nader-ziada thanks for the catch, did not realize that we are setting env vars

done, PTAL